sdk/azcore/doc.go

```diff
@@ -55,13 +55,13 @@ Template for implementing a stateful Policy:
    }
 
    // TODO: add initialization args to NewMyStatefulPolicy()
-   func NewMyStatefulPolicy() Policy {
+   func NewMyStatefulPolicy() azcore.Policy {
```

Comment:
I believe this is `pipeline.Policy` now?

Reply:
Good catch, I believe it is `policy.Policy` now